File: JAVA/L3/P1/Subject.java
package LABS.L3.P1;
import java.util.Scanner;

public class Subject {
    private String discipline;

    public Subject(String discipline) {
        this.discipline = discipline;
    }

    public String getDiscipline() {
        return discipline;
    }

    public void setDiscipline(String discipline) {
        this.discipline = discipline;
    }

    public void fill(Scanner scanner){
        discipline = scanner.nextLine();
    }

    @Override
    public String toString() {
        return "Title:" +
                "Name: " + discipline;
    }
}

File: C#/L5/Subject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace L5
{
    public class Subject : InOut
    {
        private string discipline;

        public Subject()
        {
            discipline = "C#";
        }

        public Subject(string discipline)
        {
            this.discipline = discipline;
        }

        public string getDiscipline()
        {
            return discipline;
        }

        public void setDiscipline(string discipline)
        {
            this.discipline = discipline;
        }

        public void fill()
        {
            Console.WriteLine("\nSubject\ndiscipline(string):\n");
            discipline = Console.ReadLine();
        }

        public void toString()
        {
            Console.Write("\nSubject:\n" +
                    "Discipline = " + discipline + "\n");
        }
    }

}
